src/CMakeLists.txt

```diff
@@ -824,6 +824,12 @@ if (NOT WITH_SYSTEM_ROCKSDB)
 
 endif(NOT WITH_SYSTEM_ROCKSDB)
 
+include(TestBigEndian)
```

Comment:
@cbodley could you take a look at the cmake changes? `acconfig.h` will be included by `byteorder.h` in 199a26270d3f3f07a747d577e86546e4f6b0fa6e after this change. so the this change should be backward compatible.